SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -95,10 +96,20 @@ class PositioningViewController: UIViewController, GMSMapViewDelegate, UITableVi
     let DEFAULT_SITUM_COLOR = "#283380"
     let DEFAULT_POI_NAME: String = "POI"
     let DEFAULT_BUILDING_NAME: String = "Current Building"
-    let fakeLocationsOptions = ["0º", "90º", "180º", "270º", "Create marker"]
+    let fakeLocationsOptions = [
+        "0º",
+        "90º",
+        "180º",
+        "270º",
+        NSLocalizedString("positioning.createMarker", comment: "")
+    ]
     
```

Comment:
Valora pola tua experiencia se é mellor os NSLocalizedString estén declarados na cima do ficheiro para que estén todos xuntos ou se é mellor que esten no código para ter máis contexto cando se revisen